commands/manifest/template.go

```diff
@@ -118,3 +109,45 @@ func templateCmd(c *cli.Context) error {
 
 	return nil
 }
+
+func getTemplatesManifests(m dx.Manifest) (string, error) {
```

Comment:
Since originally the variable should have been `templatedManifests`, this function right now should be just `getManifests` as it returns manifests from helm templating and the raw manifests